java/ql/src/Security/CWE/CWE-502/UnsafeDeserialization.ql

```diff
@@ -21,6 +21,39 @@ class UnsafeDeserializationConfig extends TaintTracking::Configuration {
   override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
 
   override predicate isSink(DataFlow::Node sink) { sink instanceof UnsafeDeserializationSink }
+
+  override predicate isAdditionalTaintStep(DataFlow::Node prod, DataFlow::Node succ) {
+    exists(ClassInstanceExpr cie |
```

Comment:
@smowton, is `ClassInstanceExpr` preferred over `ConstructorCall`? I find the later more readable but it seems the former is a subtype of the later.

Reply:
ConstructorCall includes `super(...)` and `this(...)`, while `ClassInstanceExpr` is strictly `new XYZ(...)`